src/BlazorUI/Bit.BlazorUI/Scripts/Callouts.ts

```
@@ -191,6 +163,33 @@ namespace BitBlazorUI {
+            callout = callout || Callouts.DEFAULT_CALLOUT;
+            const current = Callouts.current;
+
+            if (current.calloutId.length === 0) {
+                Callouts.current = callout;
+                return;
+            }
+
+            //close the previous one
+            if (callout.calloutId !== current.calloutId) {
+                const previousCallout = document.getElementById(current.calloutId);
+                previousCallout && (previousCallout.style.display = 'none');
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)